docs/architecture/adr-076-combine-spec-repo.md

```diff
@@ -0,0 +1,97 @@
+# ADR 076: Combine Spec and Tendermint Repositories 
+
+## Changelog
+
+- 2022-02-04: Initial Draft. (@tychoish)
+
+## Status
+
+Accepted.
+
+## Context
+
+While the specification for tendermint was original in the same
+repository as the Go implementation, at some point the specification
+was split from the core tendermint repository and maintained
+separately from the implementation. While this makes sense in
+promoting a conceptual separation of specification and implementation,
+in practice this separation was a premature optimization apparently
+aimed at supporting alternate implementations of tendermint. 
+
+However, maintaining a spec repo has ended up being a premature
+optimization: there are no current projects to develop alternate
+implementations of tendermint based on the common specification, and
+having separate repositories creates an ongoing maintenance burden to
+support versioning and publication.
+
+## Decision
+
+The specification repository should be merged
+
+Stakeholders including representatives from the maintainers of the
+spec, the go implementation, and the tendermint rust library, agreed
+to merge the repositories in the tendermint core dev meeting on 27
+January 2022.
+
+## Alternative Approaches
+
+The main alternative to this was *do nothing* and continue to maintain
+separate versioning for both repositories, complicating the release
+process for both repositories and creating a difficult to resolve 
+
+## Detailed Design
+
+Within the `tendermint` repository, do the following to merge the
+repositories keeping history: 
+
+```bash
+git remote add spec git@github.com:tendermint/spec.git
+git fetch spec
+git checkout -b spec-master spec/master
+mkdir spec
+git ls-tree -z --name-only HEAD | xargs -0 -I {} git mv {} subdir/
+git commit -m "spec: organize specification prior to merge"
+git checkout -b spec-merge-mainline origin/master
+git merge --allow-unrelated-histories spec-master
+```
+
+This branch should be merged into the `tendermint/tendermint`
+repository as a normal branch. This commit can also be backported to
+the 0.35 branch, if needed.
+
+Then, make a final change to the specification repository adding a
+redirect to the README and marking the repository as archived. All
+issues should be recreated in the `tendermint/tendermint` repository.
```

Comment:
I believe we can transfer the issues across: https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository 

This will maintain the links so that documents referencing the issues etc. won't lose these useful info.

Reply:
nice! lets do that.